MentoringHomeWork/HW4/Task2/sum/sum.cs

```diff
@@ -0,0 +1,60 @@
+namespace Task2.sum;
+
+public class Sum
+{
+    public void Task2()
+    {
+        var result = 0;
+        Getnumber();
+        CheckSum();
+        return;
+
+
+        void Getnumber()
+        {
+            Console.WriteLine("Please enter the first number: ");
+
+            if (int.TryParse(Console.ReadLine(), out var number))
+            {
+                Console.WriteLine("Please enter the second number: ");
+
+                if (int.TryParse(Console.ReadLine(), out var number2))
+                {
+                    result = number + number2;
+                }
+                else
+                {
+                    Console.WriteLine("Error, incorrect input for the second number. Please try again.");
+                    Task2(); 
+                }
+            }
+            else
+            {
+                Console.WriteLine("Error, incorrect input for the first number. Please try again.");
+                Task2(); 
+            }
+        }
+
+        void CheckSum()
+        {
```

Comment:
![image](https://github.com/user-attachments/assets/fdb1105d-ff4d-40fe-9c58-358b954332ab)
is it what you expect?

Reply:
I dont have this issue